.circleci/config.yml

```diff
@@ -224,10 +224,10 @@ jobs:
       - checkout
       - restore_cache: *restore_cache
       - run: *gh-config
-      - run: *install
       - run:
           name: 'Install build dependencies'
           command: |
+            npm install
```

Comment:
```suggestion
            npm install --only=prod
```
Is this needed, or no?

Reply:
what does the `--only=prod` do?

Reply:
Not needed, setting node_env to production in the compile step will take care of what we need

Reply:
@AnanyaJha It avoids installing the dev dependencies for a production build